src/content/blog/2024/fastquorum_intro.mdx

```diff
@@ -0,0 +1,215 @@
+| No       | xGen Prism (xGen cfDNA & FFPE DNA Library Prep MC v2 Kit) | Integrated DNA Technologies | Dual   | Nonrandom  | [link](https://www.idtdna.com/pages/products/next-generation-sequencing/workflow/xgen-ngs-library-preparation/dna-library-preparation/cfdna-ffpe-prep-kit)                          |
+| No       | NEBNext                                                   | New England Biosciences     | Single | Random     | [link](https://www.neb.com/en-us/products/e7874nebnext-multiplex-oligos-for-illumina-unique-dual-index-umi-adaptors-dna-set-2)                                                      |
+| No       | AML MRD                                                   | TwinStrand Biosciences      | Dual   | Random     | [link](https://twinstrandbio.com/aml-assay/)                                                                                                                                        |
+| No       | Mutagenesis                                               | TwinStrand Biosciences      | Dual   | Random     | [link](https://twinstrandbio.com/mutagenesis-assay/)                                                                                                                                |
+| No       | UMI Adapter System                                        | Twist Biosciences           | Dual   | Random     | [link](https://www.twistbioscience.com/products/ngs/library-preparation/twist-umi-adapter-system)                                                                                   |
+
+# fgbio: the Fulcrum Genomics Bioinformatics toolkit
+
+That brings us to [fgbio][fgbio-github-link].
+
+Like popular Bioinformatic toolkits like samtools, the `GATK`, and `bedtools`, `fgbio` is a collection of command line tools to analyze primary Genomics data developed by [Fulcrum Genomics][fulcrum-genomics-link].
+Since its conception in 2015, `fgbio` has been downloaded from [Bioconda][fgbio-bioconda-link] over three-hundred thousand times, and we use it extensively with our clients at [Fulcrum Genomics][fulcrum-genomics-link].
+It’s been downloaded mostly by us, but hopefully a few of you have too.
+
+        <Image src={bioconda_download} alt="Chart of Bioconda downloads placing fgbio with three-hundred-thousand downloads" height="300" />
+
+        <Image src={big_deal} alt="Will Ferrell in Anchorman meme with caption '>300K downloads on BioConda, we are kind of a big deal'" height="300" />
+
+The [fgbio toolkit has a wide variety of available tools][fgbio-list-of-tools-link], but particularly relevant for the [`nf-core/fastquorum`][nf-core-fastquorum-link] pipeline are the tools for working with read-level data containing these unique molecular indexes.
```

Comment:
Could maybe give 2-3 extra examples of what other diverse things the fgbio tookit has/does (at least as a reader I would be curious, to see if it's worth it before clicking the link for the whole thing)